rfc/0076-subdoc-replica-reads.md

```diff
@@ -0,0 +1,183 @@
+# Meta
+
+| Field          | Value                      |
+|----------------|----------------------------|
+| RFC Name       | Sub-Document Replica Reads |
+| RFC ID         | 76                         |
+| Start Date     | 2023-06-20                 |
+| Owner          | Charles Dixon              |
+| Current Status | DRAFT                      |
+| Revision       | #1                         |
+
+# Summary
+[MB-23162](https://issues.couchbase.com/browse/MB-23162) adds support for performing subdocument read operations on replica servers.
+Replica read operations are performed against replica servers with a new ReplicaRead protocol flag.
+it also includes a new, informational, HELLO flag to be negotiated.
+
+This RFC specifies the SDK work needed to support this feature.
+All SDKs are required to implement this feature.
+
+# Motivation
+SDKs require a way to expose this new ReplicaRead feature in a way that is idiomatic to the SDK API.
```

Comment:
Adding a method to a public interface is technically a breaking change, because 3rd parties may have implemented their own mocks.  Some languages don't care (duck typing), some have ways of working around it (default interface implementation), but it should be mentioned and guarded against compile-time breaks.

Reply:
This seems kind of SDK specific, if required then I think a new language specifics section should be added and populated by SDK owners - like https://github.com/couchbaselabs/sdk-rfcs/blob/master/rfc/0030-compression.md#language-specifics